v2/googlecloud-to-googlecloud/docs/SpannerToBigQuery/README.md

```diff
@@ -0,0 +1,184 @@
+#Spanner to BigQuery Dataflow Template
+
+The [Spanner to BigQuery](/src/main/java/com/google/cloud/teleport/v2/templates/SpannerToBigQuery.java) pipeline is a
```

Comment:
It did not work. I think it's because there's no way to get src from inside docs. I'll just remove the link completely

Reply:
You can do a relative path: `../../src/`

Or you can just get the GitHub URL of the file, which is actually probably better, since the link is less likely to break.